src/components/molecules/BenefitFilter.js

```diff
@@ -10,33 +10,32 @@ export function BenefitFilter(props) {
   const handleClick = (event) => props.onFilter(event.currentTarget.id);
 
   return (
-    <div className="w-full">
-      <p className="text-text-gray-dk leading-none font-bold">{props.text}</p>
-      <div className="w-full flex flex-col md:flex-row p-2">
+    <div className="w-full py-2">
+      <p className="text-text-gray-dk leading-none text-sm">{props.text}</p>
+      <div className="w-full flex flex-col md:flex-row py-2">
         <ActionButton
           id="eligible"
           text={props.eligibleText}
-          invert={props.isSelectedEligible === false}
+          invert={true}
```

Comment:
Is there a reason you are removing this condition? Isn't it needed to identify if the filter has been selected?

Reply:
the invert is for using alternate style for the component, I got rid of the invert, only using className.

Reply:
While removing the invert you seem to also have accidently removed the functionality to indicate that the filter was selected.

Also you should completely remove the invert if the functionality of it is removed, by removing the prop and all references to it.

Reply:
There is an id attribute on each button, which will be pass to the handleClick function when clicked to determine which button was selected. The invert prop in the parent component was to use a different style of the button.